helm/templates/ingestor-statefulset.yaml

```diff
@@ -139,6 +139,14 @@ spec:
           {{- end }}
           - name: P_MODE
             value: "ingest"
+          {{- if .Values.parseable.kafkaConnector.enabled }}
+          - name: P_KAFKA_BOOTSTRAP_SERVERS
+            value: {{ .Values.parseable.kafkaConnector.bootstrapServers | quote }}
+          - name: P_KAFKA_CONSUMER_TOPICS
+            value: {{ .Values.parseable.kafkaConnector.consumerTopics | quote }}
+          - name: P_KAFKA_PARTITION_LISTENER_CONCURRENCY
+            value: {{ .Values.parseable.kafkaConnector.partitionListenerConcurrency | quote }}
+          {{- end }}
```

Comment:
```
{{- if .Values.parseable.kafkaConnector.enabled }}
  {{- range $key, $value := .Values.parseable.kafkaConnector.env }}
  - name: {{ $key }}
    value: {{ tpl $value $ | quote }}
  {{- end }}
{{- end }}
```

If we implement this way, the template will remain cleaner without hardcoding values.

Reply:
@nitisht @AdheipSingh  Can you check again ?